api/v1alpha1/datamovementmanager_types.go

```diff
@@ -20,65 +20,69 @@
 package v1alpha1
 
 import (
-	v1 "k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	DataMovementNamespace = "nnf-dm-system"
+
+	DataMovementWorkerLabel = "dm.cray.hpe.com/worker"
+)
+
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
-// RsyncTemplateSpec defines the desired state of RsyncTemplate
-type RsyncTemplateSpec struct {
+// DataMovementManagerSpec defines the desired state of DataMovementManager
+type DataMovementManagerSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
-	// Selector defines the pod selector used in scheduling the rsync nodes. This value is duplicated
-	// to the template.spec.metadata.labels to satisfy the requirements of the Rsync Daemon Set.
+	// Selector defines the pod selector used in scheduling the worker nodes. This value is duplicated
+	// to the template.spec.metadata.labels to satisfy the requirements of the worker's Daemon Set.
 	Selector metav1.LabelSelector `json:"selector"`
 
-	// Template defines the pod template that is used for the basis of the Rsync Daemon set that
-	// manages the Rsync Node Data Movement requests.
-	Template v1.PodTemplateSpec `json:"template"`
+	// Template defines the pod template that is used for the basis of the worker Daemon Set that
+	// manages the per node data movement operations.
+	Template corev1.PodTemplateSpec `json:"template"`
 
-	// Host Path defines the directory location of shared mounts on an individual rsync node.
+	// Host Path defines the directory location of shared mounts on an individual worker node.
 	HostPath string `json:"hostPath"`
 
 	// Mount Path defines the location within the container at which the Host Path volume should be mounted.
 	MountPath string `json:"mountPath"`
 
-	// Disable Lustre File Systems will disable the lustre file systems from within the generated
-	// Rsync Daemon Set. The result is an Rsync Daemon Set without any volume mounts. This is
-	// strictly used for testing and should be omitted during normal operation.
-	DisableLustreFileSystems *bool `json:"disableLustreFileSystems,omitempty"`
+	Subdomain string `json:"subdomain"`
 }
 
-// RsyncTemplateStatus defines the observed state of RsyncTemplate
-type RsyncTemplateStatus struct {
+// DataMovementManagerStatus defines the observed state of DataMovementManager
+type DataMovementManagerStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
+	Ready bool `json:"ready,omitempty"`
```

Comment:
Add a kubebuilder:printcolumn for this.  But once you add a printcol, you also have to add AGE--it no longer comes in by default.

https://github.com/NearNodeFlash/nnf-sos/blob/master/api/v1alpha1/nnf_datamovement_types.go#L143-L144

Reply:
I guess spec.monitor can be removed from NnfDataMovement when V2 is merged?

Reply:
+ printcolumn
No more spec.monitor! nnf-sos changes are coming